src/rebar_prv_dialyzer.erl

```diff
@@ -193,9 +194,20 @@ do_update_proj_plt(State, Plt, Output) ->
     end.
 
 proj_plt_files(State) ->
-    BasePltApps = base_plt_apps(State),
+    {BasePltApps, BasePltMods} =
+        case is_update_base_plt_active(State) of
+            false ->
+                {ok, BaseFiles} = plt_files(get_base_plt(State)),
+                BaseMods =
+                    lists:map(fun(File) ->
+                                      list_to_atom(filename:basename(File, code:objfile_extension()))
+                              end, BaseFiles),
+                {[], BaseMods};
+            true  ->
+                {base_plt_apps(State), get_config(State, base_plt_mods, [])}
+        end,
+
     PltApps = get_config(State, plt_extra_apps, []) ++ BasePltApps,
-    BasePltMods = get_config(State, base_plt_mods, []),
```

Comment:
Modifications I was planning to add are as following:
```
proj_plt_files(State) ->
    {BasePltApps, BasePltMods} =
        case is_update_base_plt_active(State) of
            false ->
                {ok, BaseFiles} = plt_files(get_base_plt(State)),
                BaseMods =
                    lists:map(fun(File) ->
                                      list_to_atom(filename:basename(File, code:objfile_extension()))
                              end, BaseFiles),
                {[], lists:usort(BaseMods ++ get_config(State, base_plt_mods, []))};
            true  ->
                {base_plt_apps(State), get_config(State, base_plt_mods, [])}
        end,

    PltApps = get_config(State, plt_extra_apps, []) ++ BasePltApps,
    PltMods = get_config(State, plt_extra_mods, []) ++ BasePltMods,
    Apps = proj_apps(State),
    DepApps = proj_deps(State),
get_files(State, DepApps ++ PltApps, Apps -- PltApps, PltMods, []).
```
So base_plt_mods is used in both cases. Am I missing something?